src/katello/agent/goferd/plugin.py

```diff
@@ -15,27 +15,35 @@
 The katello virtual agent.
 Provides content management APIs for pulp within the RHSM environment.
 """
+from six import PY2
+
 
 import os
 import sys
-import httplib
+
+if PY2:
```

Comment:
according to python-six docs instead of loading httplibs conditionally:

`from six import http_client as http`

didn't test it though :)